Backpack-SwiftUI/Nudger/Classes/BPKNudger.swift

```diff
@@ -79,12 +91,18 @@ public struct BPKNudger: View {
     public var body: some View {
         HStack(spacing: .md) {
             if let title {
-                VStack(alignment: .leading) {
-                    BPKText(title, style: .heading5)
-                    
-                    if let subtitle {
-                        BPKText(subtitle)
-                            .foregroundColor(.textSecondaryColor)
+                HStack(alignment: subtitle == nil ? .center : .top, spacing: .md) {
```

Comment:
The icon needs to be centred in the row, not with the title alone. I think this condition should be inverted?
from figma:  ![image](https://github.com/Skyscanner/backpack-ios/assets/13455159/975e5063-c1ca-4c3c-af0a-c621b1f4307e)